packages/dev/parcel-transformer-icon/IconTransformer.js

```diff
@@ -66,7 +66,7 @@ function template(asset, iconName, svg) {
   }
   return (
 `
-import {IconProps, IconContext, IconContextValue} from '~/src/Icon';
+import {IconProps, IconContext, IconContextValue} from '@react-spectrum/s2';
```

Comment:
```
@parcel/resolver-default: External dependency "@react-spectrum/s2" is not declared in package.json.

  react-spectrum/packages/@react-spectrum/s2/package.json:100:3
     99 |   },
  > 100 |   "dependencies": {
  >     |   ^^^^^^^^^^^^^^
    101 |     "@react-aria/i18n": "^3.11.0",
    102 |     "@react-aria/interactions": "^3.22.0",

  💡 Add "@react-spectrum/s2" as a dependency.
```

I'm not sure why it can't resolve it, given that the package.json it's referencing is the one it's trying to find. I don't think I can have an entry in the dependencies section for itself.

Reply:
could be a bug in parcel. Can you generate the import differently depending on if the icon is in s2 or custom?

Reply:
sure, can do that, though I'm making an assumption on the filepath